### PR TITLE
Update contributors.txt

### PR DESCRIPTION
**Pull request by adamshakhabov**
Signed it.

### CODE REVIEW

**Comment by mstv** [comment]
As a general rule, avoid merging the master branch, please, @adamshakhabov. Rather rebase and force-push instead.
Why have you closed this PR?

**Comment by adamshakhabov** [comment]
@mstv, I closed this PR, since I thought I made mistake and I should update contributors.txt in my first PR.

However, currently, I could not push my feature branch with #8441:
image

**Comment by mstv** [comment]
I guess, you tried to push to the GE repo instead of to your fork. In addition, the `#` character might be not allowed in branch names.

**Comment by adamshakhabov** [comment]
Hmm, yes, I tried to push to main GE repo. Ok, I will do it via fork. Thank you!

**Comment by pmiossec** [comment]
>In addition, the # character might be not allowed in branch names.

I should that I don't know if I'm the only one but I really don't like having just a number as a branch name? If that's not you that create this branch, you are unable to know what this branch is talking about without searching for the issue. Which is painful...

So, please add some words to the branch name...

File: contributors.txt [hunk 1]
```diff
@@ -136,5 +136,6 @@ YYYY/MM/DD, github id, Full name, email
 2020/07/03, gtisdelle, George Tisdelle, gwnt97(at)gmail.com
 2020/07/07, bstecisa, Borja Serrano, borjaserrano(at)gmail.com
 2020/08/02, FodderMK, Kevin Yockey, gitex{at]foddermk.com
+2020/09/09, adamshakhabov, Adam Shakhabov, theadamo86{at]gmail.com
 2020/09/13, hoborm, Mihaly Hobor, mail{{at]]hobor.xyz
```
Comment (mstv) [inline contributors.txt:140]: Standardize the replacement `"(at)"` for the `@` character in all three lines, please.

**Review by mstv — COMMENTED** [review]
So you could reopen this PR and we squash-merge it to the master.

**Review by mstv — APPROVED** [review]

**Comment by mstv** [comment]
Thank you

File: contributors.txt (hunk 1 above)
Comment (RussKie) [reply contributors.txt]: This doesn't actually matter. I did it so it is human readable, but has slightly reduce readability for email harvesters.

**Pull request merged**